### ISSUE TITLE
Grammarcells: Too long method in GrammarActionsDescriptor.java

### ISSUE THREAD

**Issue by enikao** [JetBrains/MPS-extensions#317]
When using grammar cells extensively, the method `doGetSideTransformActions()` can get too long:

````
[jetbrains.mps.make.CompilationErrorsHandler@69c09cb7] Compilation problems
[jetbrains.mps.make.CompilationErrorsHandler@69c09cb7] com\mbeddr\cpp\base\editor\GrammarActionsDescriptor.java : The code of method doGetSideTransformActions(TransformationMenuContext) is exceeding the 65535 bytes limit (line: 999)
Compilation finished: 1 errors, 0 warnings.
Error executing target jetbrains.mps.make.facets.JavaCompile.compile
````

Code in https://github.com/DSLFoundry/mbeddr.core/tree/grammarcells-issue-too-long-method

Attached the generated code.

[GrammarActionsDescriptor.java.txt](https://github.com/JetBrains/MPS-extensions/files/5151169/GrammarActionsDescriptor.java.txt)

**Comment by arimer** [comment]
@alexanderpann do you know if there was any change trageting this issue?

**Comment by alexanderpann** [comment]
No, we didn't change anything. This exception could happen anywhere. If we would fix this, I would like to see a generic solution to the problem. Maybe some code that automatically splits up long methods into multiple methods.

**Comment by alexanderpann** [comment]
The solution would be to extract [this code](http://127.0.0.1:63320/node?ref=r%3A0e2d0780-27a1-4dda-a429-65b192261fcc%28com.mbeddr.mpsutil.grammarcells.generator.template.main%40generator%29%2F7272510943426830284) in separate methods and call those methods.
